be/src/olap/tablet.cpp

```diff
@@ -1642,7 +1642,7 @@ void Tablet::_init_context_common_fields(RowsetWriterContext& context) {
 }
 
 Status Tablet::create_rowset(const RowsetMetaSharedPtr& rowset_meta, RowsetSharedPtr* rowset) {
-    return RowsetFactory::create_rowset(tablet_schema(), tablet_path(), rowset_meta, rowset);
+    return RowsetFactory::create_rowset(_schema, tablet_path(), rowset_meta, rowset);
```

Comment:
should use _max_version_schema?

```
TabletSchemaSPtr Tablet::tablet_schema() const {
    std::shared_lock wrlock(_meta_lock);
    return _max_version_schema;
}
```

Reply:
If `rowset_meta` has schema, new rowset will use tablet_schema in `rowset_meta`(after version 1.2), otherwise it MUST be old version before 1.2 which doesn't support light schema change and tablet only has one schema.